ARCs/arc-0034.md

```diff
@@ -0,0 +1,86 @@
+> Issues requiring resolution may have been discussed on various online platforms such as forums, discord, and social media networks.
+
+### Voting System
+The voting system for the xGov program will be the following:
+- Only members of xGov are eligible to participate in the voting process
+- Utilization of the Algo committed by xGov as a form of voting currency, with a 1:1 ratio of Algos to votes
+- Proposals that do not meet the minimum threshold of votes required will be removed from consideration.
+
+### Threshold 
+In order for a proposal to be approved, it is necessary for the number of votes in favor of the proposal to be proportionate to the amount of funds requested. This ensures that the allocation of funds is in line with the community's consensus and in accordance with democratic principles.
+
+> eg. 2 000 000 Algo are available to be given away as grants, 200 000 000 millions Algo are committed to the xGov Process (200 000 000 votes available):
+> - Proposal A request 100 000 Algos (5 % of the Amount available)
+> - Proposal A needs 5% of the votes (10 000 000 Votes) to go through
+
+### Life of a proposal
+The proposal process will have the same duration as the governance period. Within the 3 months, the following steps will be implemented:
+- Anyone can submit a proposal at any time. 
+- 0.5 months after the start of a governance periode, proposals submitted enter the process
+- 1 month is allocated for proposals refinement and temperature check. Proposals will be evaluated and refined by the community and xGovs.
+- 1 month is allocated for voting on proposals. The community will vote on proposals that have passed the refinement and temperature check stage.
+- 0.5 months for negotiation and claiming grants. Proposals that have been approved will be negotiated and grants will be claimed.
+
+### Submit a proposal
+In order to submit a proposals, a proposer needs to create a pull request on the following repository: <a href="">URL TBD</a>.
+
+Proposals **MUST**:
+- Follow the [template form provided](../assets/arc-0034/TemplateForm.md).
+- Be submitted to the folder `Governance_X` where X is upcomming governance periode (eg: `Governance_7`)
+- Have the status `Final` before the end of the temperature check. (1,5 Months after the start of a governance)
+
+### Voting on proposal
+At the start of the voting period (1,5 Months after the start of a governance) xGovs [ARC-33](arc-033.md) will votes on proposals through an on-chain votes.
+> Vote will refer to the PR number and the last commit hash known at the start of the voting process.
+
+### Grants calculation
+The allocation of grants will be distributed taking into consideration the funding request amounts.
+The Algorand foundation will initiate contact with the proposal's author to negotiate final terms and conditions.
+
+## Rationale
+The current status of the proposal process includes the following elements:
+
+- Proposals will be submitted off-chain.
```

Comment:
Proposals should be linked on-chain to prevent reliance on GitHub. Perhaps the proposal's text can be fed into a hash function (SHA512, 256, etc.) and the result put on-chain in an application for that proposal. Maybe even xGov voting could involve transactions sent to that proposal's app id.